src/StoredEvents/Repositories/EloquentStoredEventRepository.php

```diff
@@ -83,8 +87,20 @@ public function persist(ShouldBeStored $event, string $uuid = null): StoredEvent
 
         $createdAt = Carbon::now();
 
+        try {
+            $reflectionClass = new ReflectionClass(get_class($event));
+        } catch (ReflectionException) {
+            throw new InvalidStoredEvent();
+        }
+
+        if ($serializerAttribute = $reflectionClass->getAttributes(EventSerializerAttribute::class)[0] ?? null) {
+            $serializerClass = ($serializerAttribute->newInstance())->serializerClass;
+        } else {
```

Comment:
We really dislike `else` statements. Could you refactor this way by putting this line before the if statement?

```
$serializerClass = EventSerializer::class
```

Reply:
Sure!